deploy.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ "${1:-}" != "testing" && "${1:-}" != "production" ]]; then
+    echo "Please provide an environment as the first argument - either \"testing\" or \"production\"!"
+    echo "You may additionally pass in a tag suffix as second parameter that will be appended to today's date in the git and docker tags."
+    echo
+    echo "For example,"
+    echo "$ ./deploy.sh testing bugfix"
+    echo "will create tags like \"deploy-testing-2023-Aug-05-bugfix\"."
+    exit 1
+fi
+
+env="${1:-}"
+tag_suffix="${2:-}"
+if [[ -n "$tag_suffix" ]]; then
+    # add leading dash
+    tag_suffix=${tag_suffix/#/-}
+fi
+
+# Tag the currently checked-out revision with in GitHub
+date=$(date +%Y-%b-%d)
+tag="deploy-${env}-${date}${tag_suffix}"
+echo "Tagging version as $tag in git."
+git tag -a $tag -m "Deployment to ${env}" && git push origin $tag
+
+# Build the image for the Django app
+docker compose --env-file .env.${env} build
+
+# Export the images
+docker save cpmonitor:${env} -o cpmonitor.tar
+docker save klimaschutzmonitor-dbeaver -o klimaschutzmonitor-dbeaver.tar
+
+# Copy the images, the compose files, the certificate renewal cron job and the reverse proxy settings to the server
+scp -C -r cpmonitor.tar klimaschutzmonitor-dbeaver.tar docker-compose.yml crontab reload-cert.sh docker/reverseproxy/ monitoring@monitoring.localzero.net:/tmp/
+
+# Login to the server and execute everything that follows there
+ssh -tt lzm /bin/bash << EOF
+set -euo pipefail
+
+# Import the images into Docker on the server
+docker load -i /tmp/cpmonitor.tar
+docker load -i /tmp/klimaschutzmonitor-dbeaver.tar
+
+# Tag the images with the current date in case we want to roll back,
+# as well as with the environment you're deploying to (to prevent affecting the other environment)
+docker tag cpmonitor:${env} cpmonitor:${date}${tag_suffix}
```

Comment:
It's not relevant but should we use the complete ${env}-${date}${tag_suffix} as a tag. Then it's consistent with the branch name and completely unique.